src/server.ts

```diff
@@ -65,6 +65,7 @@ Sentry.init({
     // Automatically instrument Node.js libraries and frameworks.
     ...Sentry.autoDiscoverNodePerformanceMonitoringIntegrations(),
   ],
+  release: process.env.CIBOARD_SERVER_GIT_COMMIT,
```

Comment:
What will be when it runs locally?

Reply:
"Custom build on ${hostname}/${username}" or simpel "Custom build" will be enough.

Reply:
I've changed it to `custom-build` in case the commit is missing.